Kong/docs.konghq.com
# Feat: Convert navbar to dark theme

### Description

Converting navbar to dark theme and using new logos. 

I left the blue headers as-is mostly, and just had them fade into the navbar - I think that looks pretty good for the in-between stage.

I have asked the design team for what they expect from the expanded menu as well - leaving it as white for now, until they respond.

@fabianrbzg there's couple of things I couldn't figure out here, if you could take a look:
* Why the hover/interact state for the search bar looks... not so great. But I don't know how to find the source of the variable it's using for the hover state - looks like we're pulling it from somewhere else?
* Some issues with the mobile nav (see video) - can't figure out why the slideout overlaps with the nav while it's sliding:

https://github.com/Kong/docs.konghq.com/assets/54370747/29846a70-9f93-47b5-ac91-15d2d094df3d

https://konghq.atlassian.net/browse/DOCU-3681

### Testing instructions

Preview link: check a bunch of pages. For example:

https://deploy-preview-7022--kongdocs.netlify.app/
https://deploy-preview-7022--kongdocs.netlify.app/hub/
https://deploy-preview-7022--kongdocs.netlify.app/api/
https://deploy-preview-7022--kongdocs.netlify.app/konnect/

### Checklist 


For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.
